packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2977,7 +2977,7 @@ class _WidgetInspectorState extends State<WidgetInspector> with WidgetsBindingOb
           excludeFromSemantics: true,
           child: IgnorePointer(ignoring: isSelectMode, key: _ignorePointerKey, child: widget.child),
         ),
-        _InspectorOverlay(selection: selection),
+        Positioned.fill(child: _InspectorOverlay(selection: selection)),
```

Comment:
Why was this added?

Reply:
Without this the `Stack` tries to take up as much space as possible in the parent since `_InspectorOverlay` wants to fill the screen, which causes render errors in situations where the parent of `WidgetInspector` has an unbounded dimension. Adding `Positioned.fill(...)` effectively forces the `Stack` to only size itself based on the child of `WidgetInspector`.